src/components/scenes/RequestScene.js

```diff
@@ -347,8 +347,14 @@ export class RequestComponent extends React.PureComponent<Props, State> {
                 <QrCode data={this.state.encodedURI} size={this.state.qrCodeContainerHeight - theme.rem(1)} />
               )}
             </View>
-            <RightChevronButton paddingRem={[0, 0, 0, 0]} onPress={this.handleAddressBlockExplorer} text={s.strings.request_qr_your_receiving_wallet_address} />
-            <EdgeText style={styles.publicAddressText}>{requestAddress}</EdgeText>
+            <TouchableOpacity onPress={this.handleAddressBlockExplorer}>
```

Comment:
If you are wrapping both of them, then this `RightChevronButton` should just be an icon and text, and not a button.
Otherwise, you can just make the `EdgeText` tappable.